examples/extract_data/src/extract_data_login.py

```diff
@@ -0,0 +1,72 @@
+import os
+from airtop import AsyncAirtop, SessionConfigV1
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# Setup environment variables
+api_key = os.getenv("AIRTOP_API_KEY")
+profile_id = os.getenv("profile_id")
+
+if not api_key:
+    print("Error: AIRTOP_API_KEY environment variable must be set.")
+    exit(1)
+
+TARGET_URL = ""
+EXTRACT_DATA_PROMPT = """
+This browser is open to a page that lists available job roles for software engineers in San Francisco. Please provide 10 job roles that appear to be posted by the AI-related companies.
+Return the result as a json object following this structure:
+{
+    "companies":[
+        {
+            "companyName":"",
+            "jobTitle":"",
+            "location":"",
+            "salary":"",
+        }
+    ],
+}
+"""
+
+try:
+    # Initialize AirTop client
+    client = AsyncAirtop(api_key=api_key)
+
+    # Create a session configuration
+    if profile_id != "":
+        configuration = SessionConfigV1(
```

Comment:
If no profile id is provided, you should provide a way for them to login and press Enter. Here are 2 examples on how this is done in our TS examples:
- https://github.com/airtop-ai/examples-typescript/blob/29646b819df3fb765a78b78572a5a1db74919564/examples/yc-batch-company-employees/src/cli/yc-company-employees.cli.ts#L74
- https://github.com/airtop-ai/recipe-prompt-content/blob/main/index.ts#L111

This PR should be providing the Python parity of [this example](https://github.com/airtop-ai/recipe-prompt-content/tree/main) though I like that your is a bit more generic by not mentioning glassdoor.

Note: I know you will create a separate PR for the Fern docs update but to clarify the idea is that [this existing Fern doc](https://docs.airtop.ai/recipes/recipes/extract-data-from-a-webpage-requiring-login) for the TS example is updated to that there are toggles to get the TS / Python code depending on the user preference like we do on our guides: https://docs.airtop.ai/guides/how-to/creating-a-session#how-to-create-a-session

![image](https://github.com/user-attachments/assets/a25b994b-65b4-41b3-aa47-aec6e980abc0)

Reply:
I tried to create a similar example in Python but there are a few methods and other properties that don't exist in Airtop python library as they exist in TS.
I'll look into it and I'll try to reproduce the exact same example then

Reply:
Pushed a translation as best as I could